app/src/components/Tabs/Activity/activity-tab.tsx

```diff
@@ -89,23 +64,34 @@ const ActivityTab: FC<ActivityTabProps> = ({
   const [selectedMethodology, setSelectedMethodology] = useState("");
   const [isUnavailableChecked, setIsChecked] = useState<boolean>(false);
   const [hasActivityData, setHasActivityData] = useState<boolean>(false);
-  const [methodology, setMethodology] = useState<MethodologyValues>();
+  const [methodology, setMethodology] = useState<Methodology>();
+
+  function getMethodologies() {
+    const refNumberWithScope = refNumber + "." + (filteredScope.scope || 1);
+    const methodologies =
+      MANUAL_INPUT_HIERARCHY[refNumberWithScope]?.methodologies || [];
```

Comment:
I would recommend to extract `MANUAL_INPUT_HIERARCHY[refNumberWithScope]` into a variable and doing a null check beforehand, along with maybe doing object destructuring and then using the default operator (`??`).